_posts/2020-28-08-Open-Science-Saves-Lives.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Invitation to co-sign “Open Science Saves Lives”?"
+author: "Lonni Besançon"
+date: "2020-28-08"
+layout: post
+categories: community
+---
+
+Guest post by [Lonni Besançon](https://twitter.com/lonnibesancon)
+
+
+Since the beginning of the COVID-19 pandemic, a large number of low-quality studies have been conducted and shared as preprints or fast-tracked for publication in journals, wasting research funds and decreasing efficiency of the research process. In some cases, questionable claims made by these studies have been amplified by the media and resulted in dire consequences on public policy, research, and associated public health, such as the high-profile case of Gautret et al. (2020), which resulted in Hydroxychloroquine being promoted as a cure for COVID-19 despite extremely weak and questionable evidence of its efficiency. In this paper, we attempt to identify the root causes of these problems by analysing data related to each stage of the research process, and find that 27% of COVID-19-related retractions were due to flawed designs or analyses, 43% of fast-tracked articles, accepted in a day or less, had potential conflicts of interest between authors and many unreviewed COVID-19 preprints were widely covered by news. In line with these results, we argue that Open Science Practices have the potential to vastly improve the quality of future research on COVID-19 -- for example by improving study designs through pre-registration, ensuring reliability through open code and data, and exposing conflicts of interest through open peer review -- and that public education about Open Science practices (e.g., preprints) will be necessary to prevent further loss of scientific progress and life moving forward. 
+
+The article is available here: https://www.biorxiv.org/content/10.1101/2020.08.13.249847v1 
+
+While we may not all be working in medical research, we believe that our findings and recommendations extend to all fields of research and thereby invite you to co-sign this article to show that Open Science principles are supported by a large population of scientists. Signatures will be added to the paper as supplementary material
+
+The process of co-signing this article will go as follow:
+You will have until August 31 at midnight (PDT) to co-sign.
+You have to provide an institutional email address that we will manually verify before sending you a verification email that will require action on your part.
+
+
+
+(We welcome feedback on the preprint and may make some minor changes to the text prior to submission.)
```

Comment:
Why only minor ? What if there is a very critical review that would need major changes?

Reply:
We can't ask people to sign if the manuscript changes too much